cluster/gce/gci/configure-helper.sh

```diff
@@ -1195,8 +1195,10 @@ function start-kube-addons {
     setup-addon-manifests "addons" "dns"
     local -r dns_controller_file="${dst_dir}/dns/kubedns-controller.yaml"
     local -r dns_svc_file="${dst_dir}/dns/kubedns-svc.yaml"
+    local -r dns_sa_file="${dst_dir}/dns/kubedns-sa.yaml"
```

Comment:
Alright, I'm betting I borked the add-on manager somehow.  @cjcullen @ncdc @mikedanese anything obvious?